HelloWorld/HelloWorld.cpp

```diff
@@ -11,18 +11,18 @@
 //    names to the corresponding #count of declarations.
 //
 // USAGE:
-//   clang -cc1 -load libHelloWorld.dylib -plugin hello-world '\'
+//   clang -cc1 -load build/lib/libHelloWorld.dylib -plugin hello-world '\'
```

Comment:
build` is a very specific directory - a lot of people will give it some other name.  Personally I prefer `<BUILD_DIR>` or something similar. This way it reads a bit more like: **your** build directory.

And if you don't like `<BUILD_DIR>` in the invocation, you could try this:
```
cd <BUILD_DIR>
clang -cc1 -load lib/libHelloWorld.dylib -plugin hello-world 
```

What do you reckon? I think that either way this is pretty clear, so I'm not too worried. (see also my overall comment)